### PR TITLE
5256 gtm link update with 7376

### PR DESCRIPTION
**Pull request by shondriane**
### Ticket(s)

- _[[Replace with link to Airtable ticket](https://airtable.com/appfJZShN8K4tcWHU/pagXdnDYi0tVl4u8R?HjEAY=recHbRa3Bs9DBBGID)]_

### Type of change

_Please delete any options that are not relevant_

- [ x] Bug fix (non-breaking change which fixes an issue)

### Description
Google tag was not firing on all pages. I removed the google tag manager from app.tsx and now it only exists on document.tsx. I also added a trigger for GTM tags to show history change. Now the tag is firing on all pages. 

### Screenshots
current error 
image
/image### Checklist:

_Please delete any options that are not relevant_

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings in the console
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -3,17 +3,17 @@ import '../styles/global.css';
 import { CacheProvider, EmotionCache } from '@emotion/react';
 import { CssBaseline, ThemeProvider } from '@mui/material';
 import Head from 'next/head';
-import { useRouter } from 'next/router';
-import React, { useEffect } from 'react';
-import TagManager from 'react-gtm-module';
+
+import React from 'react';
+
```
Comment (e-papanicolas) [inline pages/_app.tsx:8]: @shondriane I thought we had troubleshooted ESLint running for you, can you double check that ESLint is running, and that you are able to see the 2 errors on this page, and then by hitting save, they both auto-correct?

File: pages/_app.tsx [hunk 2]
```diff
@@ -3,17 +3,17 @@ import '../styles/global.css';
...
 
 import Footer from '../components/layout/Footer.tsx';
 import Header from '../components/layout/Header.tsx';
 import theme from '../styles/themes/theme.tsx';
 import createEmotionCache from '../utils/createEmotionCache.ts';
 
 const clientSideEmotionCache = createEmotionCache();
-const gtmId = String(process.env.NEXT_PUBLIC_GTM_ID);
+// const gtmId = String(process.env.NEXT_PUBLIC_GTM_ID);
```
Comment (e-papanicolas) [inline pages/_app.tsx:16]: Let's delete this commented out line.

File: pages/_document.tsx [hunk 3]
```diff
@@ -10,9 +10,23 @@ import createEmotionCache from '../utils/createEmotionCache.ts';
 
 export default class MyDocument extends Document {
   render() {
+    const GTM_ID = process.env.NEXT_PUBLIC_GTM_ID;
+
     return (
       <Html lang="en-US">
         <Head>
+           {/* Google Analytics */}
```
Comment (e-papanicolas) [inline pages/_document.tsx:18]: I would also try saving to see if ESLint auto-corrects this line, it looks like there is an issue with the indentation.

File: pages/_document.tsx [hunk 4]
```diff
@@ -63,21 +77,13 @@ export default class MyDocument extends Document {
...
- 
-          gtag('config', 'GTM-5SPM3GH');
-        `}
-          </Script>
-          {/* End Google Analytics */}
-
         </Head>
         <body>
+          <noscript
+            dangerouslySetInnerHTML={{
```
Comment (e-papanicolas) [inline pages/_document.tsx:83]: For this ESLint error, use the eslint-disabl marking that you can see on line 117 to allow this.

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
Please let me know about the ESLint issue! Thanks!

File: pages/_app.tsx (hunk 2 above)
Comment (shondriane) [reply pages/_app.tsx:16]: I must have deleted this already because  it's not showing up on my end.

File: pages/_document.tsx (hunk 3 above)
Comment (shondriane) [reply pages/_document.tsx:18]: Ok, will do

File: pages/_document.tsx (hunk 4 above)
Comment (shondriane) [reply pages/_document.tsx:83]: Done

**Comment by shondriane** [comment]
Hi Eleni,  the EsLint issues have been resolved. Is there another change request?

**Comment by e-papanicolas** [comment]
@shondriane nope I thought I approved/merged this the other day when we were talking about it, oops!

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**